docs/userguide/tables.md

```diff
@@ -171,8 +171,51 @@ Tablesaw supports inner and outer joins between tables.
 
 
 ## Add and remove rows
+###Adding New Row
 
+To add a new row into the table you simply need to call
+```java
+Row newRow = aTable.appendRow();
+```
+then just set the values of each column in the row
+```java
+newRow.setString("s","value1");
+newRow.setDate("d",LocalDate.now());
+```
+
+###Adding row at certain index
```

Comment:
I do not think this is a very common pattern and would prefer to leave this out of the docs. I think it is more noise than it is worth.

Reply:
i was also dubious about adding this point, but then i though it was certainly useful to me so it might be helpful to someone else too. Plus i also though it would give user idea of of how to split the tables (As the documentation already has section on combining the tables but not splitting). Anyway i will just leave this part out.